core/sched.c

```diff
@@ -70,6 +66,13 @@ const uint8_t max_threads = ARRAY_SIZE(sched_threads);
 FORCE_USED_SECTION
 const uint8_t _tcb_name_offset = offsetof(thread_t, name);
 #endif
+/** @} */
+
+volatile unsigned int sched_context_switch_request;
+volatile kernel_pid_t sched_active_pid = KERNEL_PID_UNDEF;
```

Comment:
This variable is also in the [`riot_symbol_list`](http://openocd.zylin.com/gitweb?p=openocd.git;a=blob;f=src/rtos/riot.c;h=15cbb0f8564b6253c4bbc964e7d79d5d5fbfbe72;hb=refs/heads/master#l95) and should be included in the comment block.

Reply:
Good catch, mixed it up with `sched_active_thread`. (That duplication seems odd)